js/views/modals/orderDetail/summaryTab/Summary.js

```diff
@@ -743,6 +791,46 @@ export default class extends BaseVw {
       .toggleClass('hide', this.model.get('state') !== 'PAYMENT_FINALIZED');
   }
 
+  renderProcessingError() {
+    if (!this.model.vendorProcessingError) {
+      if (this.processingError) {
+        this.processingError.remove();
+        this.processingError = null;
+      }
+
+      return;
+    }
+
+    const isBuyer = this.buyer.id === app.profile.id;
+    const state = {
+      isBuyer,
+      isModerator: !!(this.moderator && this.moderator.id),
+      isOrderCancelable: this.isOrderCancelable,
+      isModerated: !!this.moderator,
+      isCase: this.isCase,
+      // TODO todo ToDo !!! TODO todo ToDo !!! TODO todo ToDo !!!
+      // TODO todo ToDo !!! TODO todo ToDo !!! TODO todo ToDo !!!
+      // TODO todo ToDo !!! TODO todo ToDo !!! TODO todo ToDo !!!
+      // TODO: factor in escrow timeout
+      isDisputable: isBuyer &&
```

Comment:
Same as above comment about #1223. Once that one is merged, I'll need to tweak this to account for the escrow timeout.